Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.9
+
+RUN apk add --no-cache tini yarn
+
+WORKDIR /data
+COPY docker-entrypoint.sh .
+
+EXPOSE 3000
+ENTRYPOINT ["/sbin/tini", "--", "/data/docker-entrypoint.sh"]
```

Comment:
Since the script is so simple, wouldn't it be better to just add the commands to the Dockerfile.